scripts/support/clone-remote-database

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -euo pipefail
+start_tm=$(date '+%s')
+export TIME="%C %E"
+
+usage()
+{
+	set +xv
+	echo 1>&2 "$0 - clone a remote mysql database to a local machine for testing"
+	echo 1>&2 ""
+	echo 1>&2 "Usage: $0 [-l <LOCAL_DB>] [-r <REMOTE_DB>] REMOTE_HOST"
+	echo 1>&2 "REMOTE_HOST -- name or ip of remote host (passed to ssh)"
+	echo 1>&2 "LOCAL_DB -- name of new database to create locally (default: 'openboxes')"
+	echo 1>&2 "REMOTE_DB -- name of database to clone from REMOTE_HOST (default: 'openboxes')"
+	echo 1>&2 ""
+	echo 1>&2 "Use with care -- this script will destroy \$LOCAL_DB if it exists!!"
```

Comment:
I would require a force argument if the database exists locally.

Reply:
Good point. Fixed in e4861fa5f.